site/ro.njk

```diff
@@ -0,0 +1,161 @@
+---
+layout: default
+meta_title: O mișcare pentru stoparea pandemiei COVID-19
+meta_description: Guvernele noastre ne dezamăgesc, așa că trebuie sa acționăm rapid! Fă-ți treaba și stai naibii acasă!
+body_class: home
+title: '#StayTheFuckHome'
+lang: ro
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">O mișcare pentru stoparea pandemiei COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Guvernele noastre nu reușesc să prevină răspândirea 
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                și să controleze pandemia <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19</a>.
+                Reacțiile încete, politicile de calmare a publicului și impulsul lor de a stabiliza economia îi 
+                împiedică să ia măsuri care să protejeze milioane de oameni de această boală. Este timpul ca noi, 
+                în calitate de cetățeni ai acestui pământ să acționăm acum și să ne facem fiecare datoria în lupta cu COVID-19.
+                <br /><br />
+                S-o spunem răspicat: <strong>Stai naibii acasă!</strong>
+            </p>
+
+            <h2 class="text-3xl">Manifestul Auto-Carantinei</h2>
+
+            <p>
+                În lipsa unor tratamente bine studiate și a unui vaccin viabil pentru care va trebui să așteptăm cel 
+                puțin încă un an, singura metodă eficientă de a menține epidemia de coronavirus sub control e de a-i 
+                oferi virusului cât mai puține șanse de a se răspândi. Următoarea listă de acțiuni ce pot fi 
+                implementate, ordonată de la cele mai ușoare la cele mai eficiente în lupta împotriva pandemiei, ar 
+                trebui să servească ca un set de sugestii pentru oamenii care doresc să se alăture mișcării atunci 
+                când cei responsabili continuă să le neglijeze.
```

Comment:
Please change "responsabili" to "iresposabili" because otherwise it doe not make sense. :)

Reply:
Done.